manifests/repo/debian.pp

```diff
@@ -16,7 +16,7 @@
   apt::source { 'jenkins':
     location => $location,
     release  => 'binary/',
-    repos    => '',
+    repos    => undef,
```

Comment:
If this is undef, can't we simply leave it out?
```suggestion
```

Reply:
In either case, that would result in `main` being used?? (The module default).  I'm guessing this isn't the intention.

Reply:
No, the intention is to get an empty string in the generated file. Won't an explicit undef do that?

Reply:
No, `undef` means "use whatever is defined" elsewhere I think. So it ends up being `main`.